Base/res/html/misc/filter.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Filter</title>
+    <style>
+        .image-box {
+          background-image: url(old-computer.png);
+          height: 240px;
+          width: 350px;
+          background-size: cover;
+          display: flex;
+          align-items: center;
+          justify-content: center;
+          margin: 5px;
+          border-radius: 50%;
+        }
+
+        .backdrop-box {
+          background-color: rgba(255, 255, 255, 0.1);
+          width: 50%;
+          height: 50%;
+          display: flex;
+          justify-content: center;
+          align-items: center;
+          position: relative;
+          top: -72%;
+          left: 26%;
+          font-size: 1.2em;
+        }
```

Comment:
Nitpick: I'd remove the `background-color` color here (since that was just to demo the backdrop-filter), and also rename the classes in this file to remove the references to backdrops.